### PR TITLE
sample:MINIBUILD-607-file picker and display the returned content

### PR DESCRIPTION
**Pull request by Abhijit-majumder**
# Description
This is a sample application PR. 
I created a file uploader page where there is a `<input type="file"/>` and a table to show all the files that are chosen,
Once chosen files can be deleted from the table.

Added file picker element
Added route for file picker element
Added unit test

## Links
https://jira.rakuten-it.com/jira/browse/MINIBUILD-607

# Checklist
- [X] I wrote/updated tests for new/changed code.
- [X] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [X] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

File: js-miniapp-sample/src/pages/file-upload.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {rows.map((row) => (
+                <TableRow key={row.name}>
+                  <TableCell component="th" scope="row">
+                    {row.name}
+                  </TableCell>
+                  <TableCell align="left">{row.type}</TableCell>
+                  <TableCell align="right">{row.size}</TableCell>
```
Comment (minh-rakuten) [inline js-miniapp-sample/src/pages/file-upload.js:81]: Should format the display number. i.e Change `22162 KB` into `22,162 KB`.

**Review by minh-rakuten — COMMENTED** [review]
In Android, tap on Choose file or Browse nothing happens.
This is the error log when open File Upload section:

```
0.chunk.js:86889 Warning: Failed prop type: Invalid prop `className` of type `array` supplied to `ForwardRef(CardContent)`, expected `string`.
    in ForwardRef(CardContent) (created by WithStyles(ForwardRef(CardContent)))
    in WithStyles(ForwardRef(CardContent)) (at file-upload.js:60)
    in div (created by ForwardRef(Paper))
    in ForwardRef(Paper) (created by WithStyles(ForwardRef(Paper)))
    in WithStyles(ForwardRef(Paper)) (created by ForwardRef(Card))
    in ForwardRef(Card) (created by WithStyles(ForwardRef(Card)))
    in WithStyles(ForwardRef(Card)) (at file-upload.js:59)
    in FileUploader (created by Context.Consumer)
    in Route (at home.js:98)
    in Switch (at home.js:93)
    in div (created by ForwardRef(Container))
    in ForwardRef(Container) (created by WithStyles(ForwardRef(Container)))
    in WithStyles(ForwardRef(Container)) (at home.js:92)
    in main (at home.js:83)
    in Router (created by BrowserRouter)
    in BrowserRouter (at home.js:76)
    in Home (at App.js:23)
    in div (at App.js:22)
    in ThemeProvider (at App.js:21)
    in Provider (at App.js:20)
    in App (at src/index.js:12)
    in StrictMode (at src/index.js:11)
```

File: js-miniapp-sample/src/pages/file-upload.js (hunk 1 above)
Comment (Abhijit-majumder) [reply js-miniapp-sample/src/pages/file-upload.js:81]: @minh-rakuten 
For the file picker to work in android there is customization required for the web-view as per the below discussion.
https://stackoverflow.com/questions/51704560/android-file-chooser-not-calling-from-android-webview/51704638

**Comment by Abhijit-majumder** [comment]
> In Android, tap on Choose file or Browse nothing happens.
> This is the error log when open File Upload section:
> 
> ```
> 0.chunk.js:86889 Warning: Failed prop type: Invalid prop `className` of type `array` supplied to `ForwardRef(CardContent)`, expected `string`.
>     in ForwardRef(CardContent) (created by WithStyles(ForwardRef(CardContent)))
>     in WithStyles(ForwardRef(CardContent)) (at file-upload.js:60)
>     in div (created by ForwardRef(Paper))
>     in ForwardRef(Paper) (created by WithStyles(ForwardRef(Paper)))
>     in WithStyles(ForwardRef(Paper)) (created by ForwardRef(Card))
>     in ForwardRef(Card) (created by WithStyles(ForwardRef(Card)))
>     in WithStyles(ForwardRef(Card)) (at file-upload.js:59)
>     in FileUploader (created by Context.Consumer)
>     in Route (at home.js:98)
>     in Switch (at home.js:93)
>     in div (created by ForwardRef(Container))
>     in ForwardRef(Container) (created by WithStyles(ForwardRef(Container)))
>     in WithStyles(ForwardRef(Container)) (at home.js:92)
>     in main (at home.js:83)
>     in Router (created by BrowserRouter)
>     in BrowserRouter (at home.js:76)
>     in Home (at App.js:23)
>     in div (at App.js:22)
>     in ThemeProvider (at App.js:21)
>     in Provider (at App.js:20)
>     in App (at src/index.js:12)
>     in StrictMode (at src/index.js:11)
> ```

The error is fixed now! Please check!

File: js-miniapp-sample/src/pages/file-upload.js (hunk 1 above)
Comment (minh-rakuten) [reply js-miniapp-sample/src/pages/file-upload.js:81]: thanks, noted.

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Review by minh-rakuten — APPROVED** [review]
Nice 👍

**Review by nitish-a-jain — APPROVED** [review]
LGTM 👍 
Though with new additions, we should be adding in code based on Typescript instead.

**Pull request merged**